src/Tutorial/PatientBrowser/index.js

```diff
@@ -0,0 +1,16 @@
+var express = require('express');
```

Comment:
I don't think this file is actually needed. Look at how it is done in https://github.com/microsoft/fhir-server-samples/tree/master/src/FhirDashboardJS. You can just use `http-server` to serve static files. No need for express.